app/Models/Concerns/HasFilters.php

```diff
@@ -432,44 +435,14 @@ protected function filterRaces(Builder $query, string $value = null): void
      */
     protected function filterLocations(Builder $query, string $value = null): void
     {
-        $key = 'location_id';
-        if ($this->filterOption('exclude')) {
-            $query->where(function ($subquery) use ($key) {
-                return $subquery->where(
-                    $this->getTable() . '.' . $key,
-                    '!=',
-                    $this->filterValue
-                )->orWhereNull($this->getTable() . '.' . $key);
-            });
-
-            return;
-        } elseif ($this->filterOption('children')) {
+        if ($this->filterOption('children')) {
             $location = Location::find($value);
```

Comment:
If I call the page with `location_id=123123123`, the page crashes. Same with race, org, and family. Bots constantly send invalid data to forms, and it would spam me if we don't ignore invalid values ;-) You can simply do `if (empty($location)) { return }`